packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -65,6 +66,25 @@ Please make sure you are running "shorebird init" from within your Flutter proje
       return ExitCode.software.code;
     }
 
+    final orgs = await codePushClientWrapper.getOrganizationMemberships();
+    if (orgs.isEmpty) {
+      logger.err(
+        '''You do not have any organizations. This should never happen. Please contact us on Discord or send us an email at contact@shorebird.dev.''',
```

Comment:
That method on the backend could also do this log. 🤷

Reply:
It probably makes sense to do both, yeah